connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/RootResource.java

```diff
@@ -36,6 +37,7 @@ public RootResource(Herder herder) {
 
     @GET
     @Path("/")
+    @Operation(summary = "Get details about the Connect cluster")
```

Comment:
It's not just about the cluster, it's also info about this particular server.
```suggestion
    @Operation(summary = "Get details about this Connect worker and the id of the Kafka cluster it is connected to")
```